prime-router/src/main/kotlin/metadata/LivdLookup.kt

```diff
@@ -0,0 +1,218 @@
+package gov.cdc.prime.router.metadata
+
+import gov.cdc.prime.router.Metadata
+
+// this will be used by the LivdMappers and the CustomFHIRFunction
+// /need to move most of LivdMappers in here
+// will need to move the tests to look at this as well
+// keep the library the way it was then use the context to use the library to pass in custom functions
+object LivdLookup {
+    /**
+     * @param testPerformedCode used for additional filtering of the test info in case we are dealing with tests that
+     * check for more than on type of disease
+     * @param processingModeCode will flag the data as test data which will make it ignore any test devices
+     * @param deviceId the ID of the test device to lookup LIVD information by
+     * @param equipmentModelId the ID of the equipment model to lookup LIVD information by
+     * @param testKitNameId the ID of the test kit to lookup LIVD information by
+     * @param equipmentModelName the name of the equipment model to lookup LIVD information by
+     * @param tableColumn the name of the table column to lookup LIVD information on
+     * @param tableRef allows us to pass in the reference to the table, just used for testing purposes
+     * @return a possible String? value based on the lookup
+     */
+    fun find(
+        testPerformedCode: String?,
+        processingModeCode: String?,
+        deviceId: String?,
+        equipmentModelId: String?,
+        testKitNameId: String?,
+        equipmentModelName: String?,
+        tableColumn: String,
+        tableRef: LookupTable? = Metadata.getInstance().findLookupTable(name = "LIVD-SARS-CoV-2")
```

Comment:
This is the second time you use the table name. Refactor to define it in one place.